zerver/lib/actions.py

```diff
@@ -1471,6 +1471,8 @@ def create_user_messages(message: Message,
             um.flags |= UserMessage.flags.mentioned
         if um.user_profile_id in ids_with_alert_words:
             um.flags |= UserMessage.flags.has_alert_word
+        if message.recipient.type in [1, 3]:
```

Comment:
This should check against `Recipient.HUDDLE`, etc.  Never user numerical constants this way -- it makes for code that's hard to read (and often risks bugs where the numbers change).